pvlib/pvsystem.py

```diff
@@ -119,10 +123,11 @@ def __init__(self,
                  modules_per_string=1, strings_per_inverter=1,
                  inverter=None, inverter_parameters=None,
                  racking_model='open_rack_cell_glassback',
-                 name=None,
+                 name=None, model_dc='pvwatts',
                  **kwargs):
 
         self.name = name
+        self.model_dc = model_dc
```

Comment:
One possibility here is to specify `model_dc` directly as a callable in both the "built in" pvlib model cases and the user-defined model case, instead of a string or a callable, respectively. I like this idea, but it may seem more opaque to some users.